src/stream/src/executor/exchange/permit.rs

```diff
@@ -128,7 +128,7 @@ impl Sender {
             Message::Chunk(c) => {
                 let card = c.cardinality().clamp(1, self.max_chunk_permits);
                 if card == self.max_chunk_permits {
-                    tracing::warn!(cardinality = c.cardinality(), "large chunk in exchange")
+                    tracing::debug!(cardinality = c.cardinality(), "large chunk in exchange")
```

Comment:
`debug` means that we are not able to see it in production anymore. Is it still expected?

Reply:
I’m okay with keeping it at WARN if it aids in troubleshooting.

Reply:
Let's keep it at `WARN`, I think it suggests some issue on the user side if this warning keeps showing up. It can affect our backpressure mechanism if chunks are too large.